createWebNavigator.js

```diff
@@ -1,31 +1,37 @@
-import _ from 'underscore';
+import _, { compose } from 'underscore';
 import * as React from 'react';
-import {View, Pressable} from 'react-native';
-import {useNavigationBuilder, createNavigatorFactory, StackRouter} from '@react-navigation/native';
+import { View, Pressable, Dimensions } from 'react-native';
+import { useNavigationBuilder, createNavigatorFactory, StackRouter } from '@react-navigation/native';
```

Comment:
The previous way this was is more aligned with our JS styles. Let's only include the necessary changes to show what this PR is doing?